commands/describe.go

```diff
@@ -140,17 +132,26 @@ func getFunctionURLs(gateway string, functionName string, functionNamespace stri
 
 func printFunctionDescription(funcDesc schema.FunctionDescription) {
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.TabIndent)
+	process := "<default>"
+	if funcDesc.EnvProcess != "" {
+		process = funcDesc.EnvProcess
+	}
 	fmt.Fprintln(w, "Name:\t "+funcDesc.Name)
 	fmt.Fprintln(w, "Status:\t "+funcDesc.Status)
-	fmt.Fprintln(w, "Replicas:\t "+strconv.Itoa(funcDesc.Replicas))
-	fmt.Fprintln(w, "Available replicas:\t "+strconv.Itoa(funcDesc.AvailableReplicas))
+	fmt.Fprintln(w, "Replicas:\t "+strconv.Itoa(int(funcDesc.Replicas)))
+	fmt.Fprintln(w, "Available replicas:\t "+strconv.Itoa(int(funcDesc.AvailableReplicas)))
 	fmt.Fprintln(w, "Invocations:\t "+strconv.Itoa(funcDesc.InvocationCount))
 	fmt.Fprintln(w, "Image:\t "+funcDesc.Image)
-	fmt.Fprintln(w, "Function process:\t "+funcDesc.EnvProcess)
+	fmt.Fprintln(w, "Function process:\t "+process)
 	fmt.Fprintln(w, "URL:\t "+funcDesc.URL)
 	fmt.Fprintln(w, "Async URL:\t "+funcDesc.AsyncURL)
 	printMap(w, "Labels", *funcDesc.Labels)
 	printMap(w, "Annotations", *funcDesc.Annotations)
+	printList(w, "Constraints", funcDesc.Constraints)
+	printMap(w, "Environment", funcDesc.EnvVars)
```

Comment:
How about "Environment Variables"?

Reply:
The extra length is going to force the content even wider and add a lot of empty space to the other rows, which will make it harder to read. 

It is always `env` or `environment` in yaml specs, so I think it is pretty clear what it means here.

`Environment` also aligns with the output from `kubectl describe`